### PR TITLE
Support for 7" Ampire display

### PR DESCRIPTION
**Pull request by glidero**
Support for 7" Ampire display, type AM-1280800P3TZQW-00H-AMPI, resulution: 1280x800, brightness: 1500 cd/m2, LVDS interface, 24-bit color depth

### CODE REVIEW

**Review by kedder — CHANGES_REQUESTED** [review]
Thanks a lot! This looks good to me, just remove that bak file - they are not very useful considering the file history git provides.

File: recipes-bsp/u-boot/files/openvario-7-AMPI/openvario_defconfig.bak [hunk 1]
```diff
@@ -0,0 +1,44 @@
+CONFIG_ARM=y
```
[the remark this answers is not in the document]
Comment (glidero) [reply recipes-bsp/u-boot/files/openvario-7-AMPI/openvario_defconfig.bak]: thanks, the .bak file has been deleted

**Review by kedder — APPROVED** [review]
LGTM, recommend merging with  "Squash and Merge" option.

**Review by linuxianer99 — CHANGES_REQUESTED** [review]
Please change the name to 
openvario-7-AM070
to be consistend with existing machines

**Review by linuxianer99 — APPROVED** [review]

**Pull request merged**